kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/build/revapi-config.json

```diff
@@ -22,11 +22,42 @@
       }
     }
   },
-
   "ignores": {
     "revapi": {
-    "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
-    "ignore": [
+      "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
+      "ignore": [
+        {
```

Comment:
@bxf12315 the two first blocks at "ignore" section  have to be removed, the first good one is the one that makes reference to [JBPM-9740] in "justification".

Reply:
the two sections is rest api change and swagger change's comment.